src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    public static final int DEFAULT_INITIAL_CAPACITY = 16;
+    public static final float DEFAULT_LOAD_FACTOR = 0.75f;
+    public static final int CAPACITY_INCREASER = 2;
+
+    Node<K, V>[] bucketArray = new Node[DEFAULT_INITIAL_CAPACITY];
+    private int size = 0;
 
     @Override
     public void put(K key, V value) {
+        if (size + 1 > bucketArray.length * DEFAULT_LOAD_FACTOR) {
+            resize();
+        }
 
+        int bucket = bucket(key);
+        if (bucketArray[bucket] == null) { //empty bucket
+            bucketArray[bucket] = new Node<>(key, value);
+            size++;
+        } else {
+            Node<K, V> current = bucketArray[bucket];
+            for (int i = 0; i < size; i++) {
```

Comment:
you are now inside your bucket, so you shouldn't iterate to size of whole table, i think it will be better to use while loop,